TERN.md

```diff
@@ -266,6 +299,11 @@ The pixel produces a JSON object for the `input` of the form:
 }
 ```
 
+`.writeAdvertisementData(input)` should return `true` if the browser accepts this JSON object and
+`false` otherwise. There may be instances where the user is willing to accept a first-party cookie
```

Comment:
So I'm imagining a very binary control in the browser: to accept interest-based targeting or not, across all advertisers. This does leak information here, but only one bit, which I don't see as privacy-violating. Maybe I'm missing something: what would be the cross-site tracking mechanism if this were the case?

With more granular controls, say blocking ads for individual advertisers, I can clearly see how this would enable cross-site tracking. Two questions on this:

1) Is this a typical use case? I'm thinking about mechanisms at present, such as ad blockers. While they do allow granular control, I wonder how frequently this capability is used, rather than rejecting tracking as a principle.

2) A potential way out of this would be to allow the return value here not to go back to the page, but could be fed into a report for the Aggregate Reporting API. But then the browser needs to participate in reporting even if it doesn't want tracking... which is a little weird, though not functionally different from how opting out works today (e.g. NextRoll just writes the third-party cookie with an identifier of `OPT_OUT`, which indicates to not process data from the browser). Do you think a mechanism like this is reasonable?

We added this because trackability is a pretty important metric for advertisers. It lets them know what percentage of their audience is even reachable, and lets them set appropriate budgets based on that.

Reply:
Ah sure.  I was thinking of some of the [User Interface Controls](https://github.com/WICG/turtledove#user-interface-controls) like "Block this advertiser".

> We added this because trackability is a pretty important metric for advertisers.

Sure, and this is indeed the kind of thing that should be trackable in aggregate, which is right in line with your metrics goal.  I'm just expressing skepticism of reporting on it in a user-by-user context.

Reply:
Ok, if we're cool with a browser opting out of tracking but still participating in reporting, I'll try to get a commit out later today that incorporates this feedback.